### PR TITLE
Example of SparkStreaming to H2OFrame

### PR DESCRIPTION
**Pull request by jakubhava**

### CODE REVIEW

File: examples/streamingDemo/src/main/scala/ai/h2o/RandomEvent.scala [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package ai.h2o
+
+/**
+  * Created by markc on 6/7/16.
+  */
+case class RandomEvent(
+                      datetime: String,
```
Comment (vpatryshev) [inline examples/streamingDemo/src/main/scala/ai/h2o/RandomEvent.scala]: A typical case of String typing, no?

File: examples/streamingDemo/src/main/scala/ai/h2o/PipelineDemo.scala [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    // this is used to implicitly convert an RDD to a DataFrame.
+    import sqlContext.implicits._
+
+    import hc._
+    import hc.implicits._
+
+    val lines = ssc.socketTextStream("localhost", port)
+    lines.print() // useful to see some of the data stream for debugging
+    val events = lines.map(_.split(",")).map(
+      e=> RandomEvent(e(0), e(1).toDouble, e(2).toDouble)
```
Comment (vpatryshev) [inline examples/streamingDemo/src/main/scala/ai/h2o/PipelineDemo.scala]: What happens with bad data?

**Comment by mmalohlava** [comment]
Sorry, I am really confused here :)

Why this change is going into master again? It was merge already - it is part of `apps/streaming` folder.

**Comment by jakubhava** [comment]
Sorry, I missed that. I was confused that this branch was still around and I couldn't find the code in any of the usual examples directories! Closing this PR and deleting the branch

**Pull request closed without merging**